server/src/main/java/org/apache/uniffle/server/storage/LocalStorageManager.java

```diff
@@ -181,6 +181,7 @@ public Storage selectStorage(ShuffleDataFlushEvent event) {
               storage.getBasePath(), event);
         }
       } else {
+        event.setUnderStorage(storage);
```

Comment:
This is to check the storage is hdfs or localfile to put its corresponding thread pool ? Right?

Reply:
#881 already add.
```
if (event.getUnderStorage() == null) {
          event.setUnderStorage(storage);
}
```